tests/baselines/reference/asyncImportNestedYield.errors.txt

```diff
@@ -0,0 +1,16 @@
+error TS2318: Cannot find global type 'AsyncIterableIterator'.

```

Comment:
I'd fix the errors since they're not relevant to the test, then this looks good.